monkey/tests/unit_tests/monkey_island/cc/resources/test_propagation_credentials.py

```diff
@@ -21,9 +21,9 @@
     _stolen_collection,
 )
 
-ALL_CREDENTIALS_URL = PropagationCredentials.urls[0]
-CONFIGURED_CREDENTIALS_URL = urljoin(ALL_CREDENTIALS_URL, _configured_collection)
-STOLEN_CREDENTIALS_URL = urljoin(ALL_CREDENTIALS_URL, _stolen_collection)
+ALL_CREDENTIALS_URL = PropagationCredentials.urls[0].rstrip("/")
```

Comment:
It shouldn't have a trailing slash, right?

```suggestion
ALL_CREDENTIALS_URL = PropagationCredentials.urls[0]
```

Reply:
Not anymore!